framework/db/Command.php

```diff
@@ -850,4 +864,23 @@ protected function queryInternal($method, $fetchMode = null)
 
         return $result;
     }
+
+    /**
+     * Marks specified table schema to be refreshed after command execution.
+     * @param string $name name of the table, which schema should be refreshed.
+     */
+    protected function requireTableSchemaRefreshment($name)
```

Comment:
Rename it to `requireTableSchemaRefresh()` because `refreshment` has a different meaning.